tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
this technically means we don't have to `import React from 'react'` manually in all files. But I didn't wanted to refactor them all. Might run a code-mod in the future.